Showcase/Showcase/Pages/Tokens/Color/NamedColor/NamedColor+Background.swift

```diff
@@ -0,0 +1,101 @@
+            case .colorBackgroundSecondary:
+                return theme.colorBackgroundSecondary
+//            case .colorBackgroundTertiary:
+//                return theme.colorBackgroundTertiary
+            case .colorBackgroundEmphasized:
+                return theme.colorBackgroundEmphasized
+//            case .colorBackgroundBrandPrimary:
+//                return theme.colorBackgroundBrandPrimary
+//            case .colorBackgroundBrandSecondary:
+//                return theme.colorBackgroundBrandSecondary
+//            case .colorBackgroundBrandTertiary:
+//                return theme.colorBackgroundBrandTertiary
+            case .colorBackgroundStatusNeutral:
+                return theme.colorBackgroundStatusNeutral
+            case .colorBackgroundStatusNeutralOnBackgroundEmphasized:
+                return theme.colorBackgroundStatusNeutralOnBackgroundEmphasized
+            case .colorBackgroundStatusPositiveMuted:
+                return theme.colorBackgroundStatusPositiveMuted
+            case .colorBackgroundStatusPositiveMutedOnBackgroundEmphasized:
+                return theme.colorBackgroundStatusPositiveMutedOnBackgroundEmphasized
+            case .colorBackgroundStatusPositiveEmphasized:
+                return theme.colorBackgroundStatusPositiveEmphasized
+            case .colorBackgroundStatusInfoMuted:
+                return theme.colorBackgroundStatusInfoMuted
+            case .colorBackgroundStatusInfoMutedOnBackgroundEmphasized:
+                return theme.colorBackgroundStatusInfoMutedOnBackgroundEmphasized
+            case .colorBackgroundStatusInfoEmphasized:
+                return theme.colorBackgroundStatusInfoEmphasized
+            case .colorBackgroundStatusWarningMuted:
+                return theme.colorBackgroundStatusWarningMuted
+            case .colorBackgroundStatusWarningMutedOnBackgroundEmphasized:
+                return theme.colorBackgroundStatusWarningMutedOnBackgroundEmphasized
+            case .colorBackgroundStatusWarningEmphasized:
+                return theme.colorBackgroundStatusWarningEmphasized
+            case .colorBackgroundStatusNegativeMuted:
+                return theme.colorBackgroundStatusNegativeMuted
+            case .colorBackgroundStatusNegativeMutedOnBackgroundEmphasized:
+                return theme.colorBackgroundStatusNegativeMutedOnBackgroundEmphasized
+            case .colorBackgroundStatusNegativeEmphasized:
+                return theme.colorBackgroundStatusNegativeEmphasized
+//            case .colorBackgroundStatusAccentMuted:
+//                return theme.colorBackgroundStatusAccentMuted
+//            case .colorBackgroundStatusAccentMutedOnBackgroundEmphasized:
+//                return theme.colorBackgroundStatusAccentMutedOnBackgroundEmphasized
+            case .colorBackgroundStatusAccentEmphasized:
+                return theme.colorBackgroundStatusAccentEmphasized
+            }
+        }
+    }
+}
```

Comment:
Normalement tu dois avoir des warnings tels que :
- "Blanket Disable Command Violation: The disabled 'identifier_name' rule should be re-enabled before the end of the file (blanket_disable_command)"
- Blanket Disable Command Violation: The disabled 'line_length' rule should be re-enabled before the end of the file (blanket_disable_command)
- "Blanket Disable Command Violation: The disabled 'function_body_length' rule should be re-enabled before the end of the file (blanket_disable_command)"

Les warnings désactivés dans un fichier doivent être réactivés ensuite.

Il faut ajouter à la fin du fichier, après un saut de ligne, les lignes : 

```swift
// swiftlint:enable identifier_name
// swiftlint:enable line_length
// swiftlint:enable function_body_length
```